### PR TITLE
Fix for missing Data Types for MSSQL

### PR DESCRIPTION
**Pull request by mubaidr**
'NCHAR', 'SMALLDATETIME' Data Types should be treated as Strings

### CODE REVIEW

**Comment by mubaidr** [comment]
This needs to be handled in Sequelize implementation only then we rewrite this pull request.

File: src/typeMapper.js [hunk 1]
```diff
@@ -83,7 +83,8 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
       sequelizeType instanceof DATEONLY ||
       sequelizeType instanceof TIME ||
       sequelizeType instanceof BIGINT ||
-      sequelizeType instanceof DECIMAL) {
+      sequelizeType instanceof DECIMAL ||
+      ['NCHAR', 'SMALLDATETIME'].indexOf(sequelizeType) > -1) {
```
Comment (mickhansen) [inline src/typeMapper.js]: defined as strings in your sequelize model?

Comment (mubaidr) [reply src/typeMapper.js]: Sequelize does not handle these types and therefore `sequelizeType` is just a string in this case. This thing needs to be handled in Sequelize. Closing this PR.

Comment (mubaidr) [reply src/typeMapper.js]: But do need to fix this issue in mentioned in comment: https://github.com/mickhansen/graphql-sequelize/issues/611#issuecomment-439342438

**Pull request closed without merging**